static/downloadables/connect_to_server.py

```diff
@@ -88,9 +113,13 @@ def retrieveAssociationsAndClumps(refGen, traits, studyTypes, studyIDs, ethnicit
 
 
 # format the uploaded GWAS data and get the clumps from the server
-def formatGWASAndRetrieveClumps(GWASfile, GWASextension, GWASrefGen, refGen, superPop, fileHash):
+def formatGWASAndRetrieveClumps(GWASfile, userGwasBeta, GWASextension, GWASrefGen, refGen, superPop, mafCohort, fileHash, extension):
     checkInternetConnection()
 
+    # if the extension is .txt and the mafCohort is user -- Fail this is not a valid combination
+    if extension == '.txt' and mafCohort == 'user':
+        raise SystemExit('\nIn order to use the "user" option for maf cohort, you must have uploaded a vcf. Please select a different maf cohort option. \n\n')
```

Comment:
Perhaps change the error message to:
"In order to use the "user" option for maf cohort, you must upload a vcf, not a txt file. Please upload a vcf instead, or select a different maf cohort option."